sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md

```diff
@@ -28,4 +28,4 @@ Azure Cognitive Services Form Recognizer is a cloud service that uses machine le
 - [Strongly-typing a recognized form](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample4_StronglyTypingARecognizedForm.md)
 - [Differentiate output models trained with and without labels](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample10_DifferentiateOutputModelsTrainedWithAndWithoutLabels.cs)
 - [Copy a custom model between Form Recognizer resources](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample7_CopyCustomModel.md)
-- [Bounding Box](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample9_BoundingBox.cs)
+- [Field Bounding Box](https://github.com/Azure/azure-sdk-for-net/blob/bb975f36f4781d97bad79495fb033cf75d84390c/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample9_FieldBoundingBox.cs)
```

Comment:
Because of the new checks in our pipelines for broken links, I added the link to this commit. After merging, I will create a new PR to update it to master

Reply:
Good approach!